docs/examples/jobs/job.yaml

```diff
@@ -0,0 +1,8 @@
+controller: job
+name: pival
+containers:
+- image: perl
+  name: pival
```

Comment:
why name again?
It is not needed if it's the same as top level name

Reply:
done